ci-operator/step-registry/rosa-regional-platform/provision/rosa-regional-platform-provision-commands.sh

```
@@ -43,6 +43,10 @@ if [[ -n "${ROSA_REGIONAL_HELM_OVERRIDE_YAML:-}" ]] && [[ -n "${ROSA_REGIONAL_HE
   OVERRIDE_ARGS+=(--provision-override-file "${ROSA_REGIONAL_HELM_VALUES_FILE}:${OVERRIDE_YAML}")
 fi
 
+# Generate AWS profiles from mounted credentials (idempotent, no-op if already set).
+# Backwards-compatible: older refs of rosa-regional-platform don't have this script.
+[[ -f ci/setup-aws-profiles.sh ]] && source ci/setup-aws-profiles.sh
```

Shellcheck (0.11.0)

[info] Not following: ci/setup-aws-profiles.sh was not specified as input (see shellcheck -x).

(SC1091)